staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go

```diff
@@ -269,11 +272,15 @@ func (ust *uniformScenarioThread) callK(k int) {
 	if k >= ust.nCalls {
 		return
 	}
+	maxWidth := float64(uint64max(ust.uc.initialSeats, ust.uc.finalSeats))
+	ust.uss.seatDemandIntegratorCheck.Add(maxWidth)
+	returnSeatDemand := func(time.Time) { ust.uss.seatDemandIntegratorCheck.Add(-maxWidth) }
```

Comment:
the `time.Time` argument doesn't seem to be used - let's remove it

Reply:
That would be an interface change.
It might even be a good interface change.
But I think it would have to be a separate PR.